### PR TITLE
[Swift] Update Swift Docker images to support 5.1.1 release

### PR DESCRIPTION
**Pull request by shahmishal**

### CODE REVIEW

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index 811e4d7..eef431d 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -23,6 +23,12 @@ swift:5.1-bionic-slim
 swift:5.1-slim
 swift:5.1-xenial
 swift:5.1-xenial-slim
+swift:5.1.1
+swift:5.1.1-bionic
+swift:5.1.1-bionic-sim
+swift:5.1.1-slim
+swift:5.1.1-xenial
+swift:5.1.1-xenial-sim
 swift:bionic
 swift:bionic-slim
 swift:latest
diff --git a/swift_latest/Dockerfile b/swift_latest/Dockerfile
index 7a654a5..728ea56 100644
--- a/swift_latest/Dockerfile
+++ b/swift_latest/Dockerfile
@@ -26,8 +26,8 @@ RUN export DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true && ap
 # uid                  Swift 5.x Release Signing Key <swift-infrastructure@swift.org
 ARG SWIFT_SIGNING_KEY=A62AE125BBBFBB96A6E042EC925CC1CCED3D1561
 ARG SWIFT_PLATFORM=ubuntu18.04
-ARG SWIFT_BRANCH=swift-5.1-release
-ARG SWIFT_VERSION=swift-5.1-RELEASE
+ARG SWIFT_BRANCH=swift-5.1.1-release
+ARG SWIFT_VERSION=swift-5.1.1-RELEASE
 ARG SWIFT_WEBROOT=https://swift.org/builds/
 
 ENV SWIFT_SIGNING_KEY=$SWIFT_SIGNING_KEY \
diff --git a/swift_slim/Dockerfile b/swift_slim/Dockerfile
index 9a528c1..d5d5fd8 100644
--- a/swift_slim/Dockerfile
+++ b/swift_slim/Dockerfile
@@ -17,8 +17,8 @@ RUN export DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true && ap
 # uid                  Swift 5.x Release Signing Key <swift-infrastructure@swift.org
 ARG SWIFT_SIGNING_KEY=A62AE125BBBFBB96A6E042EC925CC1CCED3D1561
 ARG SWIFT_PLATFORM=ubuntu18.04
-ARG SWIFT_BRANCH=swift-5.1-release
-ARG SWIFT_VERSION=swift-5.1-RELEASE
+ARG SWIFT_BRANCH=swift-5.1.1-release
+ARG SWIFT_VERSION=swift-5.1.1-RELEASE
 ARG SWIFT_WEBROOT=https://swift.org/builds/
 
 ENV SWIFT_SIGNING_KEY=$SWIFT_SIGNING_KEY \
diff --git a/swift_xenial-slim/Dockerfile b/swift_xenial-slim/Dockerfile
index a8fbc6f..644009a 100644
--- a/swift_xenial-slim/Dockerfile
+++ b/swift_xenial-slim/Dockerfile
@@ -17,8 +17,8 @@ RUN export DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true && ap
 # uid                  Swift 5.x Release Signing Key <swift-infrastructure@swift.org
 ARG SWIFT_SIGNING_KEY=A62AE125BBBFBB96A6E042EC925CC1CCED3D1561
 ARG SWIFT_PLATFORM=ubuntu16.04
-ARG SWIFT_BRANCH=swift-5.1-release
-ARG SWIFT_VERSION=swift-5.1-RELEASE
+ARG SWIFT_BRANCH=swift-5.1.1-release
+ARG SWIFT_VERSION=swift-5.1.1-RELEASE
 ARG SWIFT_WEBROOT=https://swift.org/builds/
 
 ENV SWIFT_SIGNING_KEY=$SWIFT_SIGNING_KEY \
diff --git a/swift_xenial/Dockerfile b/swift_xenial/Dockerfile
index e846da5..4bf7ea8 100644
--- a/swift_xenial/Dockerfile
+++ b/swift_xenial/Dockerfile
@@ -26,8 +26,8 @@ RUN export DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true && ap
 # uid                  Swift 5.x Release Signing Key <swift-infrastructure@swift.org
 ARG SWIFT_SIGNING_KEY=A62AE125BBBFBB96A6E042EC925CC1CCED3D1561
 ARG SWIFT_PLATFORM=ubuntu16.04
-ARG SWIFT_BRANCH=swift-5.1-release
-ARG SWIFT_VERSION=swift-5.1-RELEASE
+ARG SWIFT_BRANCH=swift-5.1.1-release
+ARG SWIFT_VERSION=swift-5.1.1-RELEASE
 ARG SWIFT_WEBROOT=https://swift.org/builds/
 
 ENV SWIFT_SIGNING_KEY=$SWIFT_SIGNING_KEY \
```

</details>

File: library/swift [hunk 1]
```diff
@@ -7,20 +7,20 @@ Maintainers: Ted Kremenek <kremenek@apple.com> (@tkremenek),
...
 
-Tags: 5.1-slim, 5.1-bionic-slim, bionic-slim, slim
-GitCommit: fd1691a1f73b68fb8c8ef6072e716892be54a3d2
+Tags: 5.1.1-slim, 5.1-slim, 5.1.1-bionic-sim, 5.1-bionic-slim, bionic-slim, slim
+GitCommit: 4ac33ae94bcd05bb0d5ae851d6be63264f0b9c27
 Directory: 5.1/ubuntu/18.04/slim
 
-Tags: 5.1-xenial-slim, xenial-slim
-GitCommit: fd1691a1f73b68fb8c8ef6072e716892be54a3d2
+Tags: 5.1.1-xenial-sim, 5.1-xenial-slim, xenial-slim
```
Comment (yosifkit) [inline library/swift]:
```suggestion
Tags: 5.1.1-xenial-slim, 5.1-xenial-slim, xenial-slim
```

:wink:

**Comment by shahmishal** [comment]
@yosifkit Thanks

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index 811e4d7..a237f64 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -23,6 +23,12 @@ swift:5.1-bionic-slim
 swift:5.1-slim
 swift:5.1-xenial
 swift:5.1-xenial-slim
+swift:5.1.1
+swift:5.1.1-bionic
+swift:5.1.1-bionic-sim
+swift:5.1.1-slim
+swift:5.1.1-xenial
+swift:5.1.1-xenial-slim
 swift:bionic
 swift:bionic-slim
 swift:latest
diff --git a/swift_latest/Dockerfile b/swift_latest/Dockerfile
index 7a654a5..728ea56 100644
--- a/swift_latest/Dockerfile
+++ b/swift_latest/Dockerfile
@@ -26,8 +26,8 @@ RUN export DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true && ap
 # uid                  Swift 5.x Release Signing Key <swift-infrastructure@swift.org
 ARG SWIFT_SIGNING_KEY=A62AE125BBBFBB96A6E042EC925CC1CCED3D1561
 ARG SWIFT_PLATFORM=ubuntu18.04
-ARG SWIFT_BRANCH=swift-5.1-release
-ARG SWIFT_VERSION=swift-5.1-RELEASE
+ARG SWIFT_BRANCH=swift-5.1.1-release
+ARG SWIFT_VERSION=swift-5.1.1-RELEASE
 ARG SWIFT_WEBROOT=https://swift.org/builds/
 
 ENV SWIFT_SIGNING_KEY=$SWIFT_SIGNING_KEY \
diff --git a/swift_slim/Dockerfile b/swift_slim/Dockerfile
index 9a528c1..d5d5fd8 100644
--- a/swift_slim/Dockerfile
+++ b/swift_slim/Dockerfile
@@ -17,8 +17,8 @@ RUN export DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true && ap
 # uid                  Swift 5.x Release Signing Key <swift-infrastructure@swift.org
 ARG SWIFT_SIGNING_KEY=A62AE125BBBFBB96A6E042EC925CC1CCED3D1561
 ARG SWIFT_PLATFORM=ubuntu18.04
-ARG SWIFT_BRANCH=swift-5.1-release
-ARG SWIFT_VERSION=swift-5.1-RELEASE
+ARG SWIFT_BRANCH=swift-5.1.1-release
+ARG SWIFT_VERSION=swift-5.1.1-RELEASE
 ARG SWIFT_WEBROOT=https://swift.org/builds/
 
 ENV SWIFT_SIGNING_KEY=$SWIFT_SIGNING_KEY \
diff --git a/swift_xenial-slim/Dockerfile b/swift_xenial-slim/Dockerfile
index a8fbc6f..644009a 100644
--- a/swift_xenial-slim/Dockerfile
+++ b/swift_xenial-slim/Dockerfile
@@ -17,8 +17,8 @@ RUN export DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true && ap
 # uid                  Swift 5.x Release Signing Key <swift-infrastructure@swift.org
 ARG SWIFT_SIGNING_KEY=A62AE125BBBFBB96A6E042EC925CC1CCED3D1561
 ARG SWIFT_PLATFORM=ubuntu16.04
-ARG SWIFT_BRANCH=swift-5.1-release
-ARG SWIFT_VERSION=swift-5.1-RELEASE
+ARG SWIFT_BRANCH=swift-5.1.1-release
+ARG SWIFT_VERSION=swift-5.1.1-RELEASE
 ARG SWIFT_WEBROOT=https://swift.org/builds/
 
 ENV SWIFT_SIGNING_KEY=$SWIFT_SIGNING_KEY \
diff --git a/swift_xenial/Dockerfile b/swift_xenial/Dockerfile
index e846da5..4bf7ea8 100644
--- a/swift_xenial/Dockerfile
+++ b/swift_xenial/Dockerfile
@@ -26,8 +26,8 @@ RUN export DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true && ap
 # uid                  Swift 5.x Release Signing Key <swift-infrastructure@swift.org
 ARG SWIFT_SIGNING_KEY=A62AE125BBBFBB96A6E042EC925CC1CCED3D1561
 ARG SWIFT_PLATFORM=ubuntu16.04
-ARG SWIFT_BRANCH=swift-5.1-release
-ARG SWIFT_VERSION=swift-5.1-RELEASE
+ARG SWIFT_BRANCH=swift-5.1.1-release
+ARG SWIFT_VERSION=swift-5.1.1-RELEASE
 ARG SWIFT_WEBROOT=https://swift.org/builds/
 
 ENV SWIFT_SIGNING_KEY=$SWIFT_SIGNING_KEY \
```

</details>

Build test of #6767; 3b9b7209; `amd64` (`swift`):

```console

$ bashbrew build swift:5.1.1
Using bashbrew/cache:01a3a4c518f9ffda92ffecbef94c8f2456decd19d38625d44d28f972911101b4 (swift:5.1.1)
Tagging swift:5.1.1
Tagging swift:5.1
Tagging swift:5.1.1-bionic
Tagging swift:5.1-bionic
Tagging swift:bionic
Tagging swift:latest

$ test/run.sh swift:5.1.1
testing swift:5.1.1
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'swift-hello-world' [5/5]...passed


$ bashbrew build swift:5.1.1-xenial
Using bashbrew/cache:825b06adbc0bc2fa2b4fe878c5239084f374157447bfac93027512ec894b88bc (swift:5.1.1-xenial)
Tagging swift:5.1.1-xenial
Tagging swift:5.1-xenial
Tagging swift:xenial

$ test/run.sh swift:5.1.1-xenial
testing swift:5.1.1-xenial
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'swift-hello-world' [5/5]...passed


$ bashbrew build swift:5.1.1-slim
Using bashbrew/cache:75bb8d2831dd0a40eca16d4e9523f2f41710f284c51dc46e76d757ad62d68d79 (swift:5.1.1-slim)
Tagging swift:5.1.1-slim
Tagging swift:5.1-slim
Tagging swift:5.1.1-bionic-sim
Tagging swift:5.1-bionic-slim
Tagging swift:bionic-slim
Tagging swift:slim

$ test/run.sh swift:5.1.1-slim
testing swift:5.1.1-slim
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build swift:5.1.1-xenial-slim
Using bashbrew/cache:534571af90b0995cfe422a2622db2e8d137668da959132631aae7727f3516d04 (swift:5.1.1-xenial-slim)
Tagging swift:5.1.1-xenial-slim
Tagging swift:5.1-xenial-slim
Tagging swift:xenial-slim

$ test/run.sh swift:5.1.1-xenial-slim
testing swift:5.1.1-xenial-slim
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build swift:5.0.3
Using bashbrew/cache:ed28b3f013380bd5112ffa81bd07f779b5bb41840c148925fba944248a3d8d70 (swift:5.0.3)
Tagging swift:5.0.3
Tagging swift:5.0
Tagging swift:5.0.3-bionic
Tagging swift:5.0-bionic

$ test/run.sh swift:5.0.3
testing swift:5.0.3
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'swift-hello-world' [5/5]...passed


$ bashbrew build swift:5.0.3-xenial
Using bashbrew/cache:45497937366e58c5588303041c0eddeed247dbb8855253ccc3559d355da2ff77 (swift:5.0.3-xenial)
Tagging swift:5.0.3-xenial
Tagging swift:5.0-xenial

$ test/run.sh swift:5.0.3-xenial
testing swift:5.0.3-xenial
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'swift-hello-world' [5/5]...passed


$ bashbrew build swift:5.0.3-slim
Using bashbrew/cache:a2f4cd439d8718f8386d468037af232ac27a7db4092431dbecb302aa365831fc (swift:5.0.3-slim)
Tagging swift:5.0.3-slim
Tagging swift:5.0-slim
Tagging swift:5.0.3-bionic-slim
Tagging swift:5.0-bionic-slim

$ test/run.sh swift:5.0.3-slim
testing swift:5.0.3-slim
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build swift:5.0.3-xenial-slim
Using bashbrew/cache:83bafa5cdd0bac4746bbbb44e19b6707639ef632f70a908cac3ee01eb0fb522a (swift:5.0.3-xenial-slim)
Tagging swift:5.0.3-xenial-slim
Tagging swift:5.0-xenial-slim

$ test/run.sh swift:5.0.3-xenial-slim
testing swift:5.0.3-xenial-slim
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build swift:4.2.4
Using bashbrew/cache:ec8c19e2df1c501a1abd35b96c5ef197a7647a726017fdd2f313f133a4732375 (swift:4.2.4)
Tagging swift:4.2.4
Tagging swift:4.2
Tagging swift:4

$ test/run.sh swift:4.2.4
testing swift:4.2.4
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'swift-hello-world' [5/5]...passed


$ bashbrew build swift:4.1.3
Using bashbrew/cache:440a2525a92359fee79517472ad9435608f878b7f9194623e554f042014690ce (swift:4.1.3)
Tagging swift:4.1.3
Tagging swift:4.1

$ test/run.sh swift:4.1.3
testing swift:4.1.3
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'swift-hello-world' [5/5]...passed


$ bashbrew build swift:4.0.3
Using bashbrew/cache:7256be1807d4678a887673036e16a265904c28830eb9db903b7497f12c71fb43 (swift:4.0.3)
Tagging swift:4.0.3
Tagging swift:4.0

$ test/run.sh swift:4.0.3
testing swift:4.0.3
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'swift-hello-world' [5/5]...passed

```

**Pull request merged**